e2e/testdata/fn-eval/exclude-fn-config-by-default/.expected/diff.patch

```diff
@@ -1,20 +1,27 @@
+diff --git a/db/resources.yaml b/db/resources.yaml
+index ac1fd96..64ec0ee 100644
+--- a/db/resources.yaml
++++ b/db/resources.yaml
```

Comment:
Do we want to have two test cases here, one with subpkg and one without subpkg ?

Reply:
Added new cases